### DISCUSSION TITLE
Connection of wetness

### DISCUSSION THREAD

**Discussion by mahdoosh1** [TodePond/Arroost#160] [Ideas]
By connecting a source to a target with this connection, target can be fired whatsoever, but if source is fired, it acts like water, it wets target so target cant be fired.

This is not a bad idea since if this be in the arroost, it would be possble to create a computer in arroost. I know arroost is for music making, but still this is helpful to not sing an instrument if a alternative instrument is sang!

**Comment by loglot** [comment]
Wet computing

**Comment by loglot** [comment]
I looked at my inbox and saw "connection of wetness" and that caught me so off guard lmao
